internal/virtualcircuit/virtualcircuit.go

```diff
@@ -0,0 +1,53 @@
+package virtualcircuit
+
+import (
+	metal "github.com/equinix-labs/metal-go/metal/v1"
+	"github.com/equinix/metal-cli/internal/outputs"
```

Comment:
`nit` imports grouping - go native > 3rd party > internal

Reply:
IMO we can decide later (or in a separate PR, at least) if we want to enforce a particular import grouping and then get that in place across the whole project.